URLtoPebblePng.py

```diff
@@ -0,0 +1,49 @@
+import png
+import feedparser
+import itertools
+import Image
+import urllib2
+import ctypes
+from HTMLParser import HTMLParser
+from PIL import ImageEnhance
+import argparse
+
+#Get image from URL and generate pebble compliant png
+def get_pebble_png(input_url):
+    img = urllib2.urlopen(input_url)
+    localFile = open('TempImage.jpg', 'wb')
+    localFile.write(img.read())
+    localFile.close()
+
+    #Getting current aspect ratio of image
+    tempIm = Image.open('TempImage.jpg')
+    #first index is the width
+    width = tempIm.size[0]
+    #second indes is the height
+    height = tempIm.size[1]
+
+    #maintaining aspect ratio of image
+    ratio = min(144/float(width),168/float(height))
+    size = int(float(width*ratio)),int(float(height*ratio))
+
+    #resizing image to fit Pebble screen
+    im_smaller = tempIm.resize(size,Image.ANTIALIAS)
+
+    #converting to 64 color Pebble scheme and dithering image
```

Comment:
I added an option for choosing the type of Pebble that you have. If you choose Aplite, you'll get a B/W dithered image.